pkgs/applications/version-management/scriv/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, python3
```

Comment:
use `python3Packages` instead of `python3.pkgs` (This gives us some benefit regarding splicing, the nixpkgs magic for cross compilation)

Reply:
I followed the nixpkgs section [`buildPythonApplication` function](https://nixos.org/manual/nixpkgs/unstable/#python) which states:

> When packaging a Python application with `buildPythonApplication`, it should be called with `callPackage` and passed `python` or `pythonPackages` (possibly specifying an interpreter version), like this:
>
> ``` nix
> { lib, python3 }:
> 
> python3.pkgs.buildPythonApplication rec {
>   ...
> ```

Should this be changed?

Reply:
Keep it is fine, these two ought to be identical (modulo the splicing).